test/users.spec.js

```diff
@@ -11,6 +11,13 @@ const user1 = {
   email: faker.internet.email(),
   password: faker.random.alphaNumeric(10)
 };
+
+const user2 = {
+  username: faker.random.alphaNumeric(10),
```

Comment:
Consider the DRY principle in setting up your tests

Reply:
I have implemented the changes